apps/src/applab/designMode.js

```diff
@@ -273,9 +291,11 @@ designMode.parseFromLevelHtml = function(rootEl, allowDragging) {
   }
   var levelDom = $.parseHTML(Applab.levelHtml);
   var children = $(levelDom).children();
+
   children.appendTo(rootEl);
   if (allowDragging) {
-    makeDraggable(children);
+    // children are screens. make grandchildren draggable
+    makeDraggable(children.children());
```

Comment:
there may be some merge conflicts with makeUndraggable which @joshlory just added